### ISSUE TITLE
Red highlighted words all over the place

### ISSUE THREAD

**Issue by leoddd** [google/ci_edit#221]
So, I love this editor, but I have had this really annoying issue with the dictionary feature where it will mark words and commands red all over the place and I have no idea what to do about it. How can I disable the dictionary feature?
![image](https://user-images.githubusercontent.com/20242421/97924170-824b3e80-1d5f-11eb-91f3-9795ec7b5c76.png)

**Comment by dschuyler** [comment]
### Thanks for using ci_edit!

I'd like to help, please let me know if there's more I can do.

### You have options

Turn off all spell checking:
- edit `~/.ci_edit/prefs/editor.json`
- add `{"spellChecking": false}`

Add words to your personal dictionary:
- edit `~/.ci_edit/dictionaries/dictionary.user.words`
- add words, one per line. Each word added will be treated as a correctly spelled word.

Turn off spell checking for specific file types:
- for example, don't spell check ".zshrc", but do spell check other files.
- This isn't implemented as a preference setting (yet).
    - Please let me know if you'd like me to prioritize this feature.
- [waring: advanced] 
    - It can be hacked by editing `ci_edit/app/default_prefs.py` and adding
    -   `"spelling": False,`
    - into file types that should not be spell checked (see example around line 624 for "binary" files.

**Comment by leoddd** [comment]
Thank you, I honestly had no idea there was a setting for this. I browsed through the current default prefs in the code, but couldn't find a comprehensive actual list of all options anywhere I looked.
I think the way it's currently handled is plenty good then, being able to define it per-filetype would definitely be helpful but I can make do with the work-around, I wouldn't say it's a priority.

Thanks for the helpful answer!
